docs/src/main/asciidoc/user-guide/topics/transform-target-data.adoc

```diff
@@ -9,17 +9,17 @@ letter of a data value is uppercase.
 
 .Procedure
 
-. Map the fields. This can be a one-to-one mapping, a combination mapping, 
+. Map the fields. This can be a one-to-one mapping, a combination mapping,
 or a separation mapping.
 
 . In the *Mapping Details* panel, under *Sources* or under *Targets*,
```

Comment:
I'm wondering if the Mapping Details panel still has "Sources" as a heading or whether it changed to "Source" like the main panel.

Reply:
It did not. It is plural in Mapping Details.